mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -376,6 +376,12 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT axis-aligned integer box is fully loaded in the world.
 	METHOD method_35049 getRemovalReason ()Lnet/minecraft/class_1297$class_5529;
 	METHOD method_36209 onRemoved ()V
+	METHOD method_36454 getYaw ()F
+	METHOD method_36455 getPitch ()F
+	METHOD method_36456 validateAndSetYaw (F)V
```

Comment:
```suggestion
	METHOD method_36456 setYaw (F)V
```
Only your old name if there is a setter where yaw doesn't need to be validated. I recall the angles should always be wrapped.

For out-of-bounds input, are they always wrapped to within +- 180 degrees? Or how are they handled?

Reply:
Throws exception for invalid inputs. I tried to name it `setYaw`  but there's `method_5636`.

Reply:
Judging from the implementation `LivingEntity.setYaw`, `method_5636` is more like `setBodyYaw` than `setYaw`.

Reply:
Hmm maybe that's right. (silverfish and endermite override 5636 and calls both 36456 and LivingEntity.method_5636, just to note)

Reply:
Yeah, minecraft (living, at least I think) entities do have head and body yaws. Need to distinguish those.